venus/planet/theme/index.html.tmpl

```diff
@@ -54,6 +55,10 @@
 		<li><a href="<TMPL_VAR link ESCAPE="HTML">" title="<TMPL_VAR title ESCAPE="HTML">"><TMPL_VAR name></a></li>
 		</TMPL_LOOP>
 	</ul>
+	<p>Emoji via <a href="http://twitter.github.io/twemoji/">Twem❤ji</a></p>
```

Comment:
Just to be on a safe side, we should follow https://wiki.creativecommons.org/wiki/Best_practices_for_attribution.
So, I suggest the following.
This site features "Emojis" from [Twemoji](https://github.com/twitter/twemoji) by Twitter is licensed under CC-BY 4.0

Reply:
fixed.